lib/pure/strutils.nim

```diff
@@ -1774,13 +1774,14 @@ proc escape*(s: string, prefix = "\"", suffix = "\""): string {.noSideEffect,
 
 proc unescape*(s: string, prefix = "\"", suffix = "\""): string {.noSideEffect,
   rtl, extern: "nsuUnescape".} =
-  ## Unescapes a string `s`.
+  ## Unescapes a string `s` using nim's string escape rules.
```

Comment:
Not sure if that is its purpose. I imagined it to use some lowest common denominator escape rules.

Reply:
I'm not too opinionated on where this functionality belongs, but it does seem useful in stdlib. Does macros.nim sound better since it is most useful when writing macros?